plugins/russian/ranobelib.ts

```diff
@@ -104,6 +105,10 @@ class RLIB implements Plugin.PluginBase {
       }
     });
 
+    this.ui = loadedCheerio("a.header-right-menu__item")
+      .attr("href")
+      ?.replace?.(/[^0-9]/g, "");
```

Comment:
this property is alive only from when it is assigned to when app is closed. 
is it really important?

Reply:
Without it, you will not be able to open chapters and some books if you are logged in via `WebView'.